### PR TITLE
Add `lscpu` binary to container

### PR DESCRIPTION
**Pull request by jpculp**
**Description of changes:**

`lscpu` is a system utility in the `util-linux` package that the SSM
agent uses to collect instance CPU information when inventories are
enabled. The `libsmartcols` library is also included as a dependency.

**Testing done:**

- Launched `aws-ecs-1` ami with new control container set in userdata.
- Enabled the admin container via the control container's APIclient.
- Verified APIclient Exec functionality.
- Verified Bottlerocket access via `sudo sheltie`.
- Connected to admin container via ec2 instance connect.

**Additional testing done:**

Verified `AWS:InstanceDetailedInformation` was populated with CPU information after enabling inventory in AWS Systems Manager.

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -17,14 +17,24 @@ RUN \
...
+      --prefix=/opt/util-linux \
+      --without-audit \
+      --without-python \
+      --without-readline \
+      --without-systemd \
+      --without-udev \
+      --without-utempter \
     || { cat config.log; exit 1; }
+RUN make -j`nproc` lscpu
 RUN make -j`nproc` script
```
Comment (bcressey) [inline Dockerfile]: nit: could potentially combine these:
```suggestion
RUN make -j`nproc` lscpu script
```

File: Dockerfile [hunk 2]
```diff
@@ -8,7 +8,7 @@ COPY ./sdk-fetch /usr/local/bin
 
 ARG utillinux_version=2.37.4
 
-WORKDIR /opt/build
+WORKDIR $HOME/build
```
Comment (bcressey) [inline Dockerfile]: nit:
```suggestion
WORKDIR ${HOME}/build
```

**Review by bcressey — APPROVED** [review]

**Comment by jpculp** [comment]
Consolidated commands and added curly braces to `$HOME`.

**Review by cbgbt — APPROVED** [review]
:shipit:

**Pull request merged**